general-model/cosmos_client.tla

```diff
@@ -129,6 +129,8 @@ Operations == [type: {"write"}, data: Nat, region: WriteRegions, client: Clients
     (* Reads with consistency checks *)
     macro read()
     {
+        (* I check session token for consistent_prefix *)
+        when Consistency /= "consistent_prefix" \/ \A i \in WriteRegions : Data[self[1]] = prefix_token + 1; 
```

Comment:
There is no effect to use `WriteRegions`

The needed condition (and specification) is that the Invariant "ConsistentPrefix" is not true when Consistency is "eventual". And this macro seems to be not sufficient to meet this condition.